Formula/yacas.rb

```diff
@@ -18,22 +18,37 @@ class Yacas < Formula
   depends_on "cmake" => :build
   depends_on xcode: :build
 
+  on_linux do
+    depends_on "gcc"
+  end
+
+  fails_with :gcc do
+    version "6"
+    cause "needs std::string_view"
+  end
+
   def install
     cmake_args = std_cmake_args + [
       "-DENABLE_CYACAS_GUI=OFF",
       "-DENABLE_CYACAS_KERNEL=OFF",
       "-DCMAKE_C_COMPILER=#{ENV.cc}",
       "-DCMAKE_CXX_COMPILER=#{ENV.cxx}",
     ]
-    mkdir "build" do
-      system "cmake", "..", "-G", "Xcode", *cmake_args
-      ln_s "../libyacas/Release", "cyacas/libyacas_mp/Release"
-      xcodebuild "-project", "yacas.xcodeproj", "-scheme", "ALL_BUILD",
-                 "-configuration", "Release", "SYMROOT=build/cyacas/libyacas"
+    if OS.mac?
+      mkdir "build" do
+        system "cmake", "..", "-G", "Xcode", *cmake_args
+        ln_s "../libyacas/Release", "cyacas/libyacas_mp/Release"
+        xcodebuild "-project", "yacas.xcodeproj", "-scheme", "ALL_BUILD",
+                   "-configuration", "Release", "SYMROOT=build/cyacas/libyacas"
+      end
+      bin.install "build/cyacas/libyacas/Release/yacas"
+      lib.install "build/cyacas/libyacas/Release/libyacas.a",
+                  "build/cyacas/libyacas/Release/libyacas_mp.a"
```

Comment:
Can we try building with the default `Unix Makefiles` generator? Using the `Xcode` generator seems like a really odd choice here.